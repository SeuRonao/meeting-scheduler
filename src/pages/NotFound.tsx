import { useTranslation } from "react-i18next";

export default function NotFound() {
  const { t } = useTranslation("translation", { keyPrefix: "pages.NotFound" });
  return <h1 className="text-center my-3">{t("title")}</h1>;
}
