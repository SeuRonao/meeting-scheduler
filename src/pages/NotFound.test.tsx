import { create, act, ReactTestRenderer } from "react-test-renderer";
import NotFound from "./NotFound";

describe("Testing NotFound", () => {
  test("renders correctly", async () => {
    let component: ReactTestRenderer;
    await act(async () => {
      component = create(<NotFound />);
    });
    expect(component!.toJSON()).toMatchSnapshot();
  });
});
