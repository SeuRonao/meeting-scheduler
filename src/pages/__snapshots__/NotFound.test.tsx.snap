// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NotFound renders correctly 1`] = `
<h1
  className="text-center my-3"
>
  title
</h1>
`;
